src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
```

Comment:
It's a good practice to move string values to variables to avoid errors connected with typos and to promote code maintainability and readability.